mcs/class/System.Windows.Forms/System.Windows.Forms/SplitContainer.cs

```diff
@@ -395,7 +395,7 @@ public SplitContainer ()
 				if (value < panel1_min_size)
 					value = panel1_min_size;
 
-				bool updated = true;
+				bool updated;
```

Comment:
You have to explicitly assign `false` since it's a local variable. The change makes sense though.